### PR TITLE
Woody golems get woody names.

### PR DESCRIPTION
**Pull request by ghost**
I was always bothered by the wood golems having the name "Cherry" or something dumb like that, this fixes it. If you got any TREE NAMES, I'll be happy to add them

 # MAYBE IF I HAVE TIME:
unique sprite for them but don't get your hopes up

### CODE REVIEW

File: code/modules/mob/living/carbon/human/species_types/golems.dm [hunk 1]
```diff
@@ -248,7 +248,7 @@
 	heatmod = 1.5
 	info_text = "As a <span class='danger'>Wooden Golem</span>, you have plant-like traits: you take damage from extreme temperatures, can be set on fire, and have lower armor than a normal golem. You regenerate when in the light and wither in the darkness."
 	prefix = "Wooden"
-	special_names = list("Tomato", "Potato", "Broccoli", "Carrot", "Ambrosia", "Pumpkin", "Ivy", "Kudzu", "Banana", "Moss", "Flower", "Bloom", "Root", "Bark", "Glowshroom", "Petal", "Leaf", "Venus", "Sprout","Cocoa", "Strawberry", "Citrus", "Oak", "Cactus", "Pepper", "Juniper")
+	special_names = list"Bark" "Oak" "Sap" "Twig" "Branch" "Maple" "Birch" "Elm" "Basswood" "Cottonwood" "Larch" "Aspen")
```
Comment (nicbn) [inline code/modules/mob/living/carbon/human/species_types/golems.dm]: Put a `(` before `"Bark"`.

**Review by nicbn — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/carbon/human/species_types/golems.dm (hunk 1 above)
Comment (Armhulen) [reply code/modules/mob/living/carbon/human/species_types/golems.dm]: thanks

**Comment by ShadowDeath6** [comment]
cherry trees are real things though :thinking:

File: code/modules/mob/living/carbon/human/species_types/golems.dm (hunk 1 above)
Comment (ChangelingRain) [reply code/modules/mob/living/carbon/human/species_types/golems.dm]: add the following;

- Ash
- Beech
- Buckeye
- Ceder
- Chestnut
- Cypress
- Fir
- Hawthorn
- Hazel
- Hickory
- Ironwood
- Juniper
- Leaf
- Mangrove
- Palm
- Pawpaw
- Pine
- Poplar
- Redbud
- Redwood
- Sassafras
- Spruce
- Sumac
- Trunk
- Walnut
- Yew

File: code/modules/mob/living/carbon/human/species_types/golems.dm [hunk 2]
```diff
@@ -248,7 +248,7 @@
 	heatmod = 1.5
 	info_text = "As a <span class='danger'>Wooden Golem</span>, you have plant-like traits: you take damage from extreme temperatures, can be set on fire, and have lower armor than a normal golem. You regenerate when in the light and wither in the darkness."
 	prefix = "Wooden"
-	special_names = list("Tomato", "Potato", "Broccoli", "Carrot", "Ambrosia", "Pumpkin", "Ivy", "Kudzu", "Banana", "Moss", "Flower", "Bloom", "Root", "Bark", "Glowshroom", "Petal", "Leaf", "Venus", "Sprout","Cocoa", "Strawberry", "Citrus", "Oak", "Cactus", "Pepper", "Juniper")
+	special_names = list("Bark" "Oak" "Sap" "Twig" "Branch" "Maple" "Birch" "Elm" "Basswood" "Cottonwood" "Larch" "Aspen")
```
Comment (ChangelingRain) [inline code/modules/mob/living/carbon/human/species_types/golems.dm]: add the following;

- Ash
- Beech
- Buckeye
- Ceder
- Chestnut
- Cypress
- Fir
- Hawthorn
- Hazel
- Hickory
- Ironwood
- Juniper
- Leaf
- Mangrove
- Palm
- Pawpaw
- Pine
- Poplar
- Redbud
- Redwood
- Sassafras
- Spruce
- Sumac
- Trunk
- Walnut
- Yew

Comment (Armhulen) [reply code/modules/mob/living/carbon/human/species_types/golems.dm]: Thanks

**Comment by Xhuis** [comment]
ʳᵉᵐᵉᵐᵇᵉʳ ˢᵃᵖˢ

File: code/modules/mob/living/carbon/human/species_types/golems.dm [hunk 3]
```diff
@@ -248,7 +248,7 @@
 	heatmod = 1.5
 	info_text = "As a <span class='danger'>Wooden Golem</span>, you have plant-like traits: you take damage from extreme temperatures, can be set on fire, and have lower armor than a normal golem. You regenerate when in the light and wither in the darkness."
 	prefix = "Wooden"
-	special_names = list("Tomato", "Potato", "Broccoli", "Carrot", "Ambrosia", "Pumpkin", "Ivy", "Kudzu", "Banana", "Moss", "Flower", "Bloom", "Root", "Bark", "Glowshroom", "Petal", "Leaf", "Venus", "Sprout","Cocoa", "Strawberry", "Citrus", "Oak", "Cactus", "Pepper", "Juniper")
+	special_names = list("Bark", "Oak", "Sap", "Twig", "Branch", "Maple", "Birch", "Elm", "Basswood", "Cottonwood", "Larch", "Aspen", "Ash", "Beech", "Buckeye", "Cedar", "Chestnut", "Cypress", "Fir", "Hawthorn", "Hazel", "Hickory", "Ironwood", "Juniper", "Leaf", "Mangrove", "Palm", "Pawpaw", "Pine", "Poplar", "Redwood", "Redbud", "Sassafras", "Spruce", "Sumac", "Trunk", "Walnut", "Yew")
```
Comment (ChangelingRain) [inline code/modules/mob/living/carbon/human/species_types/golems.dm]: - Willow
- Catalpa

Comment (Armhulen) [reply code/modules/mob/living/carbon/human/species_types/golems.dm]: !!

**Review by XDTM — APPROVED** [review]

**Comment by PKPenguin321** [comment]
>Claims to have woody names
>Woody isn't one of the names you can actually get

**Comment by ExcessiveUseOfCobblestone** [comment]
I thought this meant they were going to have names of toys as a reference to wgw

**Comment by DoctoreMad** [comment]
One day

**Comment by uraniummeltdown** [comment]
add Woody and Morning Wood

**Comment by MoreRobustThanYou** [comment]
One day while Andy was masturbating

**Comment by ExcessiveUseOfCobblestone** [comment]
> add Woody and Morning Wood

Just put "Morning" and it'll be funny.

**Comment by XDTM** [comment]
Should we rename them treants or create treants as a subtype

**Comment by Armhulen** [comment]
Mummies are cloth golems, so.

File: code/modules/mob/living/carbon/human/species_types/golems.dm [hunk 4]
```diff
@@ -248,7 +248,7 @@
 	heatmod = 1.5
 	info_text = "As a <span class='danger'>Wooden Golem</span>, you have plant-like traits: you take damage from extreme temperatures, can be set on fire, and have lower armor than a normal golem. You regenerate when in the light and wither in the darkness."
 	prefix = "Wooden"
-	special_names = list("Tomato", "Potato", "Broccoli", "Carrot", "Ambrosia", "Pumpkin", "Ivy", "Kudzu", "Banana", "Moss", "Flower", "Bloom", "Root", "Bark", "Glowshroom", "Petal", "Leaf", "Venus", "Sprout","Cocoa", "Strawberry", "Citrus", "Oak", "Cactus", "Pepper", "Juniper")
+	special_names = list("Bark", "Willow", "Catalpa", "Woody", "Oak", "Sap", "Twig", "Branch", "Maple", "Birch", "Elm", "Basswood", "Cottonwood", "Larch", "Aspen", "Ash", "Beech", "Buckeye", "Cedar", "Chestnut", "Cypress", "Fir", "Hawthorn", "Hazel", "Hickory", "Ironwood", "Juniper", "Leaf", "Mangrove", "Palm", "Pawpaw", "Pine", "Poplar", "Redwood", "Redbud", "Sassafras", "Spruce", "Sumac", "Trunk", "Walnut", "Yew")
```
Comment (ShizCalev) [inline code/modules/mob/living/carbon/human/species_types/golems.dm]: should totally add "Morning"

File: code/modules/mob/living/carbon/human/species_types/golems.dm [hunk 5]
```diff
@@ -248,7 +248,7 @@
 	heatmod = 1.5
 	info_text = "As a <span class='danger'>Wooden Golem</span>, you have plant-like traits: you take damage from extreme temperatures, can be set on fire, and have lower armor than a normal golem. You regenerate when in the light and wither in the darkness."
 	prefix = "Wooden"
-	special_names = list("Tomato", "Potato", "Broccoli", "Carrot", "Ambrosia", "Pumpkin", "Ivy", "Kudzu", "Banana", "Moss", "Flower", "Bloom", "Root", "Bark", "Glowshroom", "Petal", "Leaf", "Venus", "Sprout","Cocoa", "Strawberry", "Citrus", "Oak", "Cactus", "Pepper", "Juniper")
```
Comment (ninjanomnom) [inline code/modules/mob/living/carbon/human/species_types/golems.dm]: Some of the old names are still good and fit this theme: Ivy, Root, Sprout

Comment (ChangelingRain) [reply code/modules/mob/living/carbon/human/species_types/golems.dm]: ivy isn't a tree!

**Comment by tgstation-server** [comment]
Your Fix/Feature pull request delta is currently below zero (-3). Maintainers may close future Feature/Tweak/Balance PRs. Fixing issues or helping to improve the codebase will raise this score.

**Pull request merged**